news/2019-05-08-community-mentorship-program-summer-2019-mentor-signups-now-open.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title: "Community Mentorship Program Spring 2019 Signups Now Open"
+date: 2019-05-08 12:00:00 +0000
+---
+Ever wanted to get into beatmapping for this game, but have no clue where to start? In that case, the Community Mentorship program might be the way to begin your mapping career!
+
+![](/wiki/shared/news/2019-01-27-community-mentorship-program-spring-2019-signups-now-open/banner.jpg)
```

Comment:
If same banner is used for every iteration or more than twice, put it [here](https://github.com/ppy/osu-wiki/tree/master/wiki/shared/news/banners) and change the references.